splash/lua_modules/splash.lua

```diff
@@ -25,7 +24,7 @@ end
 -- It is required to handle errors properly.
 --
 function Splash:jsfunc(...)
-  local func = Splash_private.jsfunc(self, ...)
+  local func = self.__private.jsfunc(self, ...)
```

Comment:
They are private because users shouldn't be able to access them; it seems that adding `__private` attribute breaks that.